distribution/feature-packs/server-feature-pack/src/main/resources/licenses/rh-sso/Select2,Apache Software License 2.0.txt

```diff
@@ -0,0 +1 @@
+Select2,GNU General Public License v2.0 only.txt
```

Comment:
Looks like select2 is actually MIT from their website at least? https://select2.github.io/